PokemonRampUp/Model/TypeElement.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TypeElement.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-02-14.
+//
+
+import Foundation
+
+// MARK: - TypeElement
+struct TypeElement: Decodable {
+    let slot: Int
```

Comment:
What is slot property used for?

Reply:
I removed will be removed it my next commit. I thought it was needed to make decoding the JSON to Pokemon model easier but, looking further into it. It is not needed.